lib/modelMaterialsCommon.js

```diff
@@ -0,0 +1,754 @@
+"use strict";
+var Cesium = require('cesium');
+var WebGLConstants = Cesium.WebGLConstants;
+var defined = Cesium.defined;
+var defaultValue = Cesium.defaultValue;
+
+module.exports = modelMaterialsCommon;
+
+function webGLConstantToGlslType(webGLValue) {
```

Comment:
Could be useful as a standalone function in a separate file...not sure if you want to move it now.